R/dsl.R

```diff
@@ -103,3 +103,47 @@ generate_random_id <- function(prefix, id_length = 20) {
   random_id <- paste(sample(letters, id_length, replace = TRUE), collapse = "")
   paste0(prefix, "-", random_id)
 }
+
+CLOSABLE_MESSAGES <- "$('.message .close')
```

Comment:
I would place this constant after uimessage definition. It would keep more consistent form of document.